frontend/app/package.json

```diff
@@ -112,6 +112,7 @@
     "unplugin-auto-import": "0.16.4",
     "unplugin-vue-components": "0.25.0",
     "unplugin-vue-define-options": "1.3.7",
+    "vh-sticky-table-header": "1.5.0",
```

Comment:
How much work is to implement this functionality as part of the application?

Reply:
In general, we avoid dependencies such as this, it is a library that is small, doesn't have traction and a single person maintains it.

We will eventually run into similar issues as with the `vue-cli-electron-builder` thing. I would prefer if we don't add this kind of dependency.

Reply:
If this is a leftover from trial an error then just remove it 😄

Reply:
was playing around, I need to modify a lot of the algorithms😛